pipelines/score_forecast.R

```diff
@@ -169,7 +184,29 @@ read_and_score_location <- function(model_run_dir,
 
   actual_data <- prep_truth_data(truth_path)
 
-  pyrenew <- arrow::read_parquet(forecast_path) |>
+  pyrenew <- arrow::read_parquet(forecast_path)
+  # Collate pyrenew forecasts by epiweek. NB default is strict weeks.
+  if (epiweekly) {
+    pyrenew <- pyrenew |>
+      filter(disease != "prop_disease_ed_visits") |>
+      group_by(disease) |>
+      group_modify(~ forecasttools::daily_to_epiweekly(.x,
+        value_col = ".value", weekly_value_name = ".value",
+        strict = strict
+      )) |>
+      ungroup() |>
+      pivot_wider(names_from = disease, values_from = .value) |>
+      mutate(prop_disease_ed_visits = Disease / (Disease + Other)) |>
+      pivot_longer(c(Disease, Other, prop_disease_ed_visits),
+        names_to = "disease",
+        values_to = ".value"
+      ) |>
+      mutate(date = epiweek_to_date(epiweek, epiyear,
+        day_of_week = day_of_week
+      ))
+  }
```

Comment:
Should something like this instead be done in `pipelines/postprocess_state_forecast.R`?

We would actually like to create epiweekly forecasts and present them, right? See https://github.com/CDCgov/pyrenew-hew/issues/79

Reply:
Yes, I think it makes sense to have an epiweekly forecast as canonical output, not just for scoring.

Related: perhaps the location-specific runs should build a location-specific hubverse table, and the overall hubverse table can then just be built via collation (right now it's built via iteration over the results in `create_hubverse_table.R`)

Reply:
I had need of this just now so I did it in 6627518